crates/sui-adapter-transactional-tests/tests/shared/upgrade.exp

```diff
@@ -12,15 +12,18 @@ created: object(107), object(108), object(109), object(110)
 written: object(106)
 
 task 3 'view-object'. lines 44-44:
-Owner: Object ID: ( fake(107) )
+Owner: Account Address ( A )
 Version: 2
 Contents: t::m::Obj {id: sui::object::UID {id: sui::object::ID {bytes: fake(110)}}}
 
 task 4 'run'. lines 46-46:
-Error: Object 0xe6ff026a903dda0711f951bab1d01161791f0f6fe94badadcb0424cf05bd591d is owned by object 0xf2c06d97f4ad9da4a8a6cfb80dd73173302be0c6c245c62279a4bc86a366f33c. Objects owned by other objects cannot be used as input arguments.
+Error: Transaction Effects Status: Move Runtime Abort. Location: sui::transfer::share_object_impl (function index 7) at offset 0, Abort Code: 0
```

Comment:
Are these expected? I see some errors here

Reply:
It is likely that the fake IDs get shuffled around.... I really which I had a different way of doing this, but the  IDs get sorted, so anytime the generation gets changed (which shouldn't happen often) these tests break

Reply:
should i change anything to the tests?

Reply:
thanks @tnowacki, I guess you refer to the whole [upgrade.exp] file. Then @joyqvq we're good to go